### PR TITLE
add qa-i18n.zh.html 

### PR DESCRIPTION
**Pull request by xgqfrms**

### CODE REVIEW

**Comment by r12a** [comment]
@xgqfrms i just discovered this in my long stack of outstanding emails.  What's the current status, do you want me to publish it?

I noticed a couple of nits while scanning the changes. I'll mention them in individual notes.

File: questions/qa-i18n-data/translations.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 var trans = { }
 
-trans.versions = ['ar','de','en','es','fr','hu','pl','pt','pt-br','ro','ru','sv','uk']
+trans.versions = ['ar','de','en','es','fr','hu','pl','pt','pt-br','ro','ru','sv','uk','zh']
```
Comment (r12a) [inline questions/qa-i18n-data/translations.js:4]: This needs to say 'zh-hans'.

File: questions/qa-i18n.zh.html [hunk 2]
```diff
@@ -0,0 +1,125 @@
+<!DOCTYPE html>
+<html lang="zh-Has">
```
Comment (r12a) [inline questions/qa-i18n.zh.html]: Typo: should say lang="zh-Hans"

File: questions/qa-i18n.zh.html [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+    f.lastSubstUpdate = { date:'2005-12-05', time:'20:40'};
+    // date and time of latest substantive changes to this document
+    f.status = 'published';
+    // should be one of draft, review, published, notreviewed or obsolete
+    f.path = '../';
+    // what you need to prepend to a URL to get to the /International directory
+    // AUTHORS AND TRANSLATORS should fill in these assignments:
+    f.thisVersion = {date:'2020-09-11', time:'09:11'};
+    // date and time of latest edits to this document/translation
+    f.contributors = 'xgqfrms';
```
Comment (r12a) [inline questions/qa-i18n.zh.html:33]: No need for this value, it's for people helping with reviews, etc. You are credited just below as translator.

File: javascript/boilerplate-text/boilerplate-zh.js [hunk 4]
```diff
@@ -0,0 +1,136 @@
+// translate the text between quotes
+// text following double slashes (such as this line) provides comments for the translator
+// words beginning with s. or f. are variables - do not disturb these ! (though you can move them)
+// where it would aid clarity, variables that occur in the text are described below
+
+var s = { }
+
+// DO NOT TRANSLATE
```
Comment (xfq) [inline javascript/boilerplate-text/boilerplate-zh.js:8]: It says "DO NOT TRANSLATE", but I see that `s.currLang` in other languages are also translated. @r12a Do you know the reason?

File: javascript/boilerplate-text/boilerplate-zh.js [hunk 5]
```diff
@@ -0,0 +1,136 @@
...
+
+s.suppStylesheets = ''
+
+s.rtlAttribute = ""
+s.ltrAttribute = ""
+s.rlm = ""
+
+
+// TRANSLATE THE FOLLOWING
+s.i18nActivityHome = "I18N 活动主页"
```
Comment (xfq) [inline javascript/boilerplate-text/boilerplate-zh.js:48]: 这里的「I18N」应译为「国际化」

File: javascript/boilerplate-text/boilerplate-zh.js [hunk 6]
```diff
@@ -0,0 +1,136 @@
...
+s.moreResourcesOfThisType = "更多本类资源"
+s.accessKeyN = '快捷键及跳转页面导航。<a href="#contentstart">跳转到正文开始。</a>'
+s.examplesInAnotherScript = "本文档包括其它语言/字符示例。"
```
Comment (xfq) [inline javascript/boilerplate-text/boilerplate-zh.js:51]:
```suggestion
s.examplesInAnotherScript = "本文档包括其它语言/文字示例。"
```

File: javascript/boilerplate-text/boilerplate-zh.js [hunk 7]
```diff
@@ -0,0 +1,136 @@
...
+s.worldMap = "世界地图"
+s.searchI18nSite = "国际化站点搜索" 
+s.translationDisclaimer = '本文档译自其它语种。如有任何偏差或错误的话，请以<a href="'+f.filename+'.en">最新英文原文</a>为准。如下所示，<a href="#copyright">原文版权</a>属于 W3C。.'
```
Comment (xfq) [inline javascript/boilerplate-text/boilerplate-zh.js:54]:
```suggestion
s.translationDisclaimer = '本文档译自其它语种。如有任何偏差或错误，请以<a href="'+f.filename+'.en">最新英文原文</a>为准。如下所示，<a href="#copyright">原文版权</a>属于 W3C。.'
```

File: javascript/boilerplate-text/boilerplate-zh.js [hunk 8]
```diff
@@ -0,0 +1,136 @@
...
+s.byTheWayLink = "其它信息"
+s.byTheWay = "其它信息"
+s.furtherReadingAlt = "深入阅读"
+s.furtherReadingLink = "深入阅读"
+s.furtherReading = "深入阅读"
+s.intendedAudience = "目标读者:"
+s.skipToAnswer = "[跳转到答案]"
+s.tellUsWhatYouThink = "告诉我们您的想法（用英语）。"
+s.sendAComment = "将您的评论发送给我们"
+s.subscribeToRSS = "订阅 RSS 提要"
```
Comment (xfq) [inline javascript/boilerplate-text/boilerplate-zh.js:97]: 感觉「RSS 提要」不是很常见，是否考虑保留英文原文，使用「RSS Feed」，或者直接使用「RSS」？

File: javascript/boilerplate-text/boilerplate-zh.js [hunk 9]
```diff
@@ -0,0 +1,136 @@
...
+s.newResourcesAlt = "每当首次发布一个新的资源时通知您。"
+s.newResources = "新的资源"
+s.homePageNewsAlt = "主页上显示的所有新闻条目。."
```
Comment (xfq) [inline javascript/boilerplate-text/boilerplate-zh.js:100]:
```suggestion
s.homePageNewsAlt = "主页上显示的所有新闻条目。"
```

File: javascript/boilerplate-text/boilerplate-zh.js [hunk 10]
```diff
@@ -0,0 +1,136 @@
...
+s.homePageNews = "主页新闻"
+s.followOurNews = "关注我们的新闻源。"
+s.sentenceDelimiter = "."
+
+s.author = "作者：" // followed by name of author(s)
+s.previousAuthors = "前作者：" // followed by name of previous author(s), before substantive changes were made by the current author
+s.modifiedBy = "修改：" // person's name appears after colon
+s.translatedBy = "翻译:"
```
Comment (xfq) [inline javascript/boilerplate-text/boilerplate-zh.js:108]:
```suggestion
s.translatedBy = "翻译："
```

File: javascript/boilerplate-text/boilerplate-zh.js [hunk 11]
```diff
@@ -0,0 +1,136 @@
...
+
+s.historyOfDocumentChanges = '有关文档更新的历史记录，请参阅主要更新的<a href="http://www.w3.org/blog/International/tag/'+f.searchString+'/">新闻源</a>，以及自2016年1月以来所有更改的 <a href="https://github.com/w3c/i18n-drafts/commits/gh-pages/'+f.directory+f.filename+'.en.html">Git 提交列表</a>。'
+s.untranslatedChanges = '<strong>注意：</strong>在此文件翻译以后<a href="'+f.filename+'.en.php">英文版本</a> 已有改变。这些更改已合并到此页面中，因此您可能会看到一些英文段落，直到翻译更新为止。'
+s.unlinkedTranslation = '<strong>警告：</strong>本文的英文原文自翻译以来进行了实质性修改，建议您在此版本更新之前阅读<a href="'+f.filename+'.en">英文版本</a>。'
+s.new="新"// placed alongside new articles at http://www.w3.org/International/articlelist 
+s.updated="已更新" // same as New
+s.translation_updated="更新译本:" // date appears after colon 
+
+s.aboutThisArticle="文章相关" // title in the right column near the top of the page
+s.status_draft="此篇文章为未经公开审阅的拟定草稿。如有信息需要及时补充或订正，请您使用页面下的网络链接及时反馈给我们。"
```
Comment (xfq) [inline javascript/boilerplate-text/boilerplate-zh.js:123]: 是否可以直接用「链接」替代「网络链接」？（下同）

File: javascript/boilerplate-text/boilerplate-zh.js [hunk 12]
```diff
@@ -0,0 +1,136 @@
...
+s.status_review="此篇文章目前正供公开审阅，请发送您的宝贵意见到 <a href='mailto:www-international@w3.org?subject=%5Breview%20feedback%5D%20$filename'>www-international@w3.org</a><span class='noprint'> (<a href='https://lists.w3.org/Archives/Public/www-international/'>订阅</a>)</span>."
+s.status_published="此篇文章已由 W3C 国际化工作组（W3C Internationalization Working Group）审阅，并经过公开审阅以尽可能的确保文章的正确性。如有信息需要及时补充或订正，请您使用页面下的网络链接及时反馈给我们。"
+s.status_notreviewed="此篇文章尚未经过公开审阅。如有信息需要及时补充或订正，请您使用页面下的网络链接及时反馈给我们。"
+s.status_obsolete="这篇文章现已过时，不会继续维护，且信息可能不准确。有关更多最新信息，请参阅<a href='http://www.w3.org/International/'>国际化活动主页</a>。"
+s.tests="测试" // used at the end of breadcrumbs for test related pages
+s.quickanswer = "简约回复" // heading
```
Comment (xfq) [inline javascript/boilerplate-text/boilerplate-zh.js:129]: answer在这里翻译成回答/答案更好一些？

例子：https://www.w3.org/International/questions/qa-html-encoding-declarations.en#quickanswer

File: questions/qa-i18n.zh.html [hunk 13]
```diff
@@ -0,0 +1,125 @@
...
+    </div>
+    <h2 id="question"><a href="#question">Question</a></h2>
+    <p class="question">术语“国际化”和“本地化”是什么意思，它们之间有什么关系？</p>
+  </section>
+  <section>
+    <h2 id="answer"><a href="#answer">Answer</a></h2>
+    <p>每个人对这些术语都有自己的首选定义。 我们在此处提供一些一般性的高级描述，说明我们倾向于在W3C国际化站点上使用这些术语的方式。</p>
+    <section>
+      <h3 id="l10n"><a href="#l10n">本地化</a></h3>
+      <p>本地化是指产品，应用程序或文档内容的<strong>适应</strong>以符合语言，文化和一个特定目标市场的其他要求（一个<span class="qterm">语言环境</span>）。</p>
```
Comment (xfq) [inline questions/qa-i18n.zh.html:76]: 这句话读着不是特别通顺，是否可以改为：

> 本地化是指为了满足特定目标市场（区域设置）的语言、文化和其他要求对产品、应用或文档内容进行的调整。

File: questions/qa-i18n.zh.html [hunk 14]
```diff
@@ -0,0 +1,125 @@
...
+      <p>本地化有时用英语写成 <span class="qterm">l10n</span>, 其中 10是<span class="qchar">l</span>和<span class="qchar">n</span>之间的英语单词中的字母数。</p>
```
Comment (xfq) [inline questions/qa-i18n.zh.html:77]: 如果10的左边加空格，右边也需要加一个空格平衡一下。

另外也需要和下面i18n的18一致。

File: questions/qa-i18n.zh.html [hunk 15]
```diff
@@ -0,0 +1,125 @@
...
+      <p>通常只将其视为用户界面和文档翻译的同义词，本地化通常是一个实质上更为复杂的问题。 它可能需要与以下内容相关的自定义：</p>
```
Comment (xfq) [inline questions/qa-i18n.zh.html:78]:
```suggestion
      <p>通常只将其视为用户界面和文档翻译的同义词，本地化通常是一个实质上更为复杂的问题。它可能需要与以下内容相关的自定义：</p>
```

File: questions/qa-i18n.zh.html [hunk 16]
```diff
@@ -0,0 +1,125 @@
...
+      <ol>
+        <li>数字，日期和时间格式</li>
```
Comment (xfq) [inline questions/qa-i18n.zh.html:80]:
```suggestion
        <li>数字、日期和时间格式</li>
```

File: questions/qa-i18n.zh.html [hunk 17]
```diff
@@ -0,0 +1,125 @@
...
+        <li>货币使用</li>
+        <li>键盘用法</li>
+        <li>整理和排序</li>
+        <li>符号，图标和颜色</li>
```
Comment (xfq) [inline questions/qa-i18n.zh.html:84]:
```suggestion
        <li>符号、图标和颜色</li>
```

File: questions/qa-i18n.zh.html [hunk 18]
```diff
@@ -0,0 +1,125 @@
...
+        <li>包含对对象，动作或思想的引用的文本和图形，在给定的文化中，它们可能会被误解或被视为不敏感。</li>
```
Comment (xfq) [inline questions/qa-i18n.zh.html:85]:
```suggestion
        <li>包含对对象、动作或思想的引用的文本和图形，在特定的文化中可能会被误解或不存在。</li>
```

这样写更自然一些。

File: questions/qa-i18n.zh.html [hunk 19]
```diff
@@ -0,0 +1,125 @@
...
+        <li>各种法律要求</li>
+        <li>还有更多东西。</li>
+      </ol>
+      <p>本地化甚至可能需要重新思考逻辑，视觉设计或表示形式，如果给定条件下的经商方式（例如会计）或公认的学习范式（例如，专注于个人与群体） 语言环境与原始文化大不相同。</p>
```
Comment (xfq) [inline questions/qa-i18n.zh.html:89]:
```suggestion
      <p>本地化甚至可能需要重新思考逻辑、视觉设计或表示形式，如果给定条件下的经商方式（例如会计）或公认的学习范式（例如，专注于个人与群体）语言环境与原始文化大不相同。</p>
```

File: questions/qa-i18n.zh.html [hunk 20]
```diff
@@ -0,0 +1,125 @@
...
+    </section>
+    <section>
+      <h3 id="i18n"><a href="#i18n">国际化</a></h3>
+      <p>国际化的定义各不相同。 这是与W3C国际化活动材料一起使用的高级工作定义。 有些人使用其他术语，例如<span class ="qterm">全球化</span>来指代相同的概念。</p>
+      <p>国际化是产品，应用程序或文档内容的设计和开发，可以针对不同文化，地区或语言的目标受众<strong>启用</strong>轻松地进行本地化。</p>
```
Comment (xfq) [inline questions/qa-i18n.zh.html:94]:
```suggestion
      <p>国际化是产品、应用程序或文档内容的设计和开发，可以针对不同文化，地区或语言的目标受众<strong>启用</strong>轻松地进行本地化。</p>
```

File: questions/qa-i18n.zh.html [hunk 21]
```diff
@@ -0,0 +1,125 @@
...
+        <li>以消除本地化或国际部署障碍的方式进行设计和开发。这包括诸如启用Unicode或确保在适当的情况下确保对旧字符编码的正确处理，注意字符串的连接，避免对用户界面字符串值的代码的依赖性等之类的事情。</li>
+        <li>提供对直到本地化才能使用的功能的支持。例如，在DTD中添加标记以支持双向文本或识别语言。或增加对垂直文本或其他非拉丁印刷功能的CSS支持。</li>
+        <li>启用代码以支持本地，区域，语言或与文化相关的首选项。通常，这涉及合并预定义的本地化数据和从现有库或用户首选项派生的功能。例如日期和时间格式，本地日历，数字格式和数字系统，列表的排序和显示，个人名称和地址形式的处理等。</li>
+        <li>将可本地化的元素与源代码或内容分开，以便可以根据需要根据用户的国际喜好加载或选择本地化的替代项。</li>
+      </ol>
+      <p>请注意，这些项目不一定包括将内容，应用程序或产品本地化为另一种语言； 它们是设计和开发实践，可以在将来轻松进行这种迁移，但是即使没有本地化，也可能具有重要的用途。</p>
+    </section>
+    <section>
+      <h3 id="value"><a href="#value">国际化的价值</a></h3>
+      <p>国际化严重影响了产品本地化的难易程度。 为全球市场而设计以语言和文化为中心的交付物显然比设计交付物要困难得多且耗时得多。 （回想一下Y2K的工作，并尝试“撤消”以“ 19xx”为基础的两个字符的年份字段。）</p>
```
Comment (xfq) [inline questions/qa-i18n.zh.html:107]:
```suggestion
      <p>国际化严重影响了产品本地化的难易程度。为全球市场而设计以语言和文化为中心的交付物显然比设计交付物要困难得多且耗时得多。（回想一下Y2K的工作，并尝试“撤消”以“ 19xx”为基础的两个字符的年份字段。）</p>
```

File: questions/qa-i18n.zh.html (hunk 2 above)
Comment (xgqfrms) [reply questions/qa-i18n.zh.html]: fixed

File: questions/qa-i18n-data/translations.js (hunk 1 above)
Comment (xgqfrms) [reply questions/qa-i18n-data/translations.js:4]: fixed

File: javascript/boilerplate-text/boilerplate-zh.js (hunk 4 above)
Comment (r12a) [reply javascript/boilerplate-text/boilerplate-zh.js:8]: When the translator receives the file, these are already pre-translated.

File: javascript/boilerplate-text/boilerplate-zh.js [hunk 22]
```diff
@@ -0,0 +1,136 @@
+// translate the text between quotes
```
Comment (r12a) [inline javascript/boilerplate-text/boilerplate-zh.js:1]: Actually, i guess that this is a new file has a -zh filename.  We should instead be using the file at https://w3c.github.io/i18n-drafts/javascript/boilerplate-text/boilerplate-zh-hans.js

**Pull request closed without merging**